pkg/controllers/federatedhpa/federatedhpa_controller.go

```diff
@@ -404,25 +404,23 @@ func (c *FHPAController) reconcileAutoscaler(ctx context.Context, hpa *autoscali
 
 func (c *FHPAController) getBindingByLabel(resourceLabel map[string]string, resourceRef autoscalingv2.CrossVersionObjectReference) (*workv1alpha2.ResourceBinding, error) {
 	if len(resourceLabel) == 0 {
-		return nil, fmt.Errorf("Target resource has no label. ")
+		return nil, errors.New("target resource has no label")
```

Comment:
What advantage does `errors.New` have over `fmt.Errorf` when used here?

Reply:
For formatting without parameters, `fmt.Errorf` equals `errors.New`. I just reduce unnecessary function calls.

Reply:
Using `errors.New` is much faster than using `fmt.Errorf`:
```
goos: windows
goarch: amd64
pkg: notebook
cpu: Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
BenchmarkFmt
BenchmarkFmt-12                 10938595               101.2 ns/op
BenchmarkErrorsNew
BenchmarkErrorsNew-12           1000000000               0.2384 ns/op
PASS
```